src/util/GameLogicUtils.hpp

```diff
@@ -281,6 +313,14 @@ namespace spy::util {
             static bool checkBabySitter(const gameplay::State &s, const gameplay::CharacterOperation &op,
                                         const spy::MatchConfig &config);
 
+            /**
+             * @brief get number of possible babysitters that could be applied
+             * @param state current state
+             * @return number of babysitters
+             */
+            static int
+            babysitterNumber(const gameplay::State &s, std::shared_ptr<const spy::gameplay::GadgetAction> action);
```

Comment:
```suggestion
            babysitterCount(const gameplay::State &s, std::shared_ptr<const spy::gameplay::GadgetAction> action);
```
(damit klar ist, dass dies eine Summe und kein index oder so ist)